config/develop/glue-job-role.yaml

```diff
@@ -1,4 +1,6 @@
 template_path: glue-job-role.yaml
```

Comment:
the `template_path` key is deprecated, you should use `template` instead.  https://docs.sceptre-project.org/dev/docs/template_handlers.html#template-handlers

Reply:
@philerooski Let's create a separate JIRA ticket to make this update, but it's an easy one to do.

Reply:
created https://sagebionetworks.jira.com/browse/ETL-111